src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.Random.cs

```diff
@@ -3,13 +3,13 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
-        {
-            byte[] bytes = new byte[8];
-            s_random.Value.NextBytes(bytes);
 
-            return $"{s_rootIdPrefix}{BitConverter.ToUInt64(bytes, 0):x}";
+        private string GenerateInstancePrefix()
```

Comment:
Presumably we don't need a separate file anymore since we got rid of random (which was implemented differently on different platforms?